dotcom-rendering/src/components/WeatherWrapper.importable.tsx

```diff
@@ -6,14 +6,18 @@ import { Weather, WeatherPlaceholder } from './Weather';
 const appendPartnerCodeToUrl = (
 	url: string | undefined,
 ): string | undefined => {
-	if (!url || !URL.canParse(url)) {
+	try {
+		if (url && URL.canParse(url)) {
```

Comment:
We may want to drop `URL.canParse`? It's an alternative to wrapping the `URL` in a `try...catch`, as described in the docs[^1], but since we _are_ using a `try...catch` here we probably don't need it any more?

If we keep it, we're saying "crash and return `undefined` for any browser that doesn't support `canParse`"? But in a lot of those cases we probably _can_ parse the URL if we just leave it up to the `URL` constructor, which has wider support[^2]?

[^1]: https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static
[^2]: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#browser_compatibility

Reply:
OK that makes sense. I'll remove `URL.canParse `